src/wallet/rpcwallet.cpp

```diff
@@ -2317,6 +2416,329 @@ UniValue listtransactions(const JSONRPCRequest& request)
+                    std::vector<std::string> csvSubRecord(TRANSACTION_CSV_FIELD_COUNT);
+                    for (unsigned int i = 0; i < wtx.tx->vpout.size(); ++i) {
+                        auto pout = wtx.tx->vpout[i];
+                        bool fIsMyOutput = pwallet->IsMine(pout.get());
+
+                        switch(pout->GetType())
+                        {
+                        case OUTPUT_DATA:
+                        {
+                            // This typically denotes the transaction fee and is the first record
+                            CTxOutData* outData = (CTxOutData*)pout.get();
+                            CAmount nFeeData;
+                            if (outData->GetCTFee(nFeeData)) {
+                                csvSubRecord[TRANSACTION_CSV_FIELD_FEE] = FormatMoney(nFeeData);
+                            }
+                            break;
+                        }
+
+                        case OUTPUT_STANDARD:
+                            if (!fIsMyOutput) {
+                                if (pout->IsZerocoinMint()) {
+                                    // Do nothing
+                                } else { // basecoin
+                                    CTxDestination dest;
+                                    if (ExtractDestination(*pout->GetPScriptPubKey(), dest)) {
+                                        csvSubRecord[TRANSACTION_CSV_FIELD_CATEGORY] = "send";
+                                        csvSubRecord[TRANSACTION_CSV_FIELD_ADDRESS] = EncodeDestination(dest, true);
+                                        csvSubRecord[TRANSACTION_CSV_FIELD_AMOUNT] = FormatMoney(pout->GetValue());
+                                        WalletTxToCsv(csvSubRecord, wtx);
+                                        csv_write.WriteRecord(csvSubRecord, true);
+                                    }
+                                }
+                            }
+                            break;
+
+                        default:
+                            // Do nothing
+                            break;
+                        } // switch
+                    } // for
+                } // if
+            }
+        }
+    }
+}
+
+
+static UniValue exporttransactions(const JSONRPCRequest& request)
+{
+    std::shared_ptr<CWallet> const wallet = GetWalletForJSONRPCRequest(request);
+    CWallet* const pwallet = wallet.get();
+
+    if (!EnsureWalletIsAvailable(pwallet, request.fHelp)) {
+        return NullUniValue;
+    }
+
+    std::string help_text {};
+    if (!IsDeprecatedRPCEnabled("accounts")) {
+        help_text = "exporttransactions (dummy filename start end categories include_watchonly)\n"
+            "\nExports transactions between 'start' date and 'end' date matching 'categories' to a CSV file.\n"
+            "Note: To export from a specified the \"account\", use this RPC with an \"account\" argument and restart\n"
```

Comment:
(As this is user facing, it needs to be fixed)
Remove "the" from "from a specified the account".  The sentence should also be re-structured to tell them to restart with depreciatedrpc, and then use the command with the account argument.

Reply:
Should be fixed now